core/pom.xml

```diff
@@ -242,6 +242,16 @@
             <artifactId>spring-web</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
```

Comment:
background: Add spring-context for testing with StaticApplicationContext

Reply:
Do you mean to use `<scope>test</scope>`?

Reply:
I guess, for the test suite to work, this would suffice -- but my comment was just meant as a review note.